deploy/helm/templates/openshift/openshift-role-binding.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.openshift.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ include "common.fullname" (tuple . "-openshift-role-binding") }}
```

Comment:
Shoud we name that this RB is for kube-state-metrics?

Reply:
it has collector name in prefix, right?
![image](https://github.com/user-attachments/assets/057ae7da-63cf-4322-aee1-39ff00f97748)